site/docs/v1/tech/lambdas/client-credentials-jwt-populate.adoc

```diff
@@ -0,0 +1,84 @@
+---
+layout: doc
+title: Client Credentials JWT Populate lambda
+description: An overview of the Client Credentials JWT Populate lambda
+---
+
+:sectnumlevels: 0
+
+== Client Credentials JWT Populate lambda
+
+If you would like to augment the claims provided in the JWT before it has been signed you can specify a lambda in the JWT configuration. This lambda will be invoked prior to the token being signed and issued to a user.
+
+When you create a new lambda using the FusionAuth UI we will provide you an empty function for you to implement. If you are using the API to create the lambda you will need to ensure your function has the following signature:
+
+[source,javascript]
+----
+function populate(jwt, recipientEntity, targetEntities, permissions) {
+  // Lambda code goes here
+}
+----
+
+This lambda must contain a function named `populate` that takes four parameters. The parameters that the lambda is passed are:
+
+* `jwt` - the claims object to be signed and return as the JWT payload
+* `permissions` - the permissions assigned to the Entity
+* `recipientEntity` - the Recipient Entity
+* `targetEntities` - the Target Entities
+
+The two FusionAuth objects are well documented here in the link:/docs/v1/tech/apis/entity-management/entity-types/[Permissions & Entity Types APIs] and link:/docs/v1/tech/apis/entity-management/entities/[Entities API] documentation. The JWT object is a JavaScript object containing the JWT payload. See link:/docs/v1/tech/oauth/tokens/[OpenID Connect & OAuth 2.0 Token].
```

Comment:
The two objects are? Assuming `recipientEntity` and `targetEntities`?

Reply:
added clarifier 👍🏻